models/victoriametrics.go

```diff
@@ -0,0 +1,128 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strconv"
+
+	config "github.com/percona/promconfig"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	vmCacheDisableEnv = "VM_CACHE_DISABLE"
+	vmTestEnableEnv   = "PERCONA_TEST_VM"
+)
+
+// VictoriaMetricsParams - defines flags and settings for victoriametrics.
+type VictoriaMetricsParams struct {
+	VMDB           []string
+	VMAlert        []string
+	Enabled        bool
+	baseConfigPath string
+}
+
+// NewVictoriaMetricsParams - returns configuration params for VictoriaMetrics
+// error can be suppressed, this func always return inited struct with default
+// params.
+func NewVictoriaMetricsParams(basePath string) (*VictoriaMetricsParams, error) {
+	vmp := &VictoriaMetricsParams{
+		VMDB:           []string{},
+		VMAlert:        []string{},
+		baseConfigPath: basePath,
+	}
+	if enabledVar := os.Getenv(vmTestEnableEnv); enabledVar != "" {
+		parsedBool, err := strconv.ParseBool(enabledVar)
+		if err != nil {
+			return vmp, fmt.Errorf("cannot parse PERCONA_TEST_VM, as bool, value: %s, err: %w", enabledVar, err)
+		}
+		vmp.Enabled = parsedBool
+	}
+
+	err := vmp.UpdateParams()
+	if err != nil {
+		return vmp, err
+	}
+
+	return vmp, nil
+}
+
+// UpdateParams - updates params for VictoriaMetrics services
+// reads configuration file and updates corresponding flags.
```

Comment:
It would be more clear if only the second line of the comment will be left with `UpdateParams ` prefix.